src/utils/subject.ts

```diff
@@ -104,16 +104,12 @@ async function getGitifySubjectForDiscussion(
     }
   }
 
-  let comments = discussion.comments.nodes;
-  let discussionUser = null;
-
-  if (comments?.length) {
-    discussionUser = getLatestDiscussionComment(comments)?.author.login;
-  }
+  const discussionUser = getLatestDiscussionComment(discussion.comments.nodes)
+    ?.author.login;
```

Comment:
Does login need a `?` too? Not sure how JS treats this.

```suggestion
    ?.author?.login;
```

Reply:
I believe this is ok.  if we get an author object thats not null, then it _should_ contain the login attribute